recipes/libigl/all/conanfile.py

```diff
@@ -0,0 +1,102 @@
+import os
+import stat
+import shutil
+from conans import ConanFile, tools, CMake, AutoToolsBuildEnvironment
+
+
+class LibiglConan(ConanFile):
+    name = "libigl"
+    description = ("A Massively Spiffy Yet Delicately Unobtrusive Compression Library "
+                  "(Also Free, Not to Mention Unencumbered by Patents)")
+    topics = ("conan", "libigl", "geometry", "matrices", "algorithms")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libigl.github.io/"
+    license = "MPL2"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package"
+    requires = ("eigen/3.3.9")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+ 
+        tools.replace_in_file(self._source_subfolder + "/CMakeLists.txt",
+                                  "project(libigl)",
+                                  """project(libigl)
+# libIGL must find the Eigen3 library imported by conan
+# otherwise it downloads a own copy from internet
+find_package(Eigen3 REQUIRED) """)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+
+        if self.options.shared:
+            cmake.definitions["LIBIGL_USE_STATIC_LIBRARY"] = "OFF"
+        else:
+            cmake.definitions["LIBIGL_USE_STATIC_LIBRARY"] = "ON"
```

Comment:
```suggestion
        cmake.definitions["LIBIGL_USE_STATIC_LIBRARY"] = not self.options.shared
```

CMake supports boolean directly, don't need to stringy to ON/OFF.